src/home_assistant.rs

```diff
@@ -89,8 +106,6 @@ impl Config {
             self.power("p_to_user", "Power from Grid")?,
             self.power("p_to_grid", "Power to Grid")?,
             self.power("p_eps", "Active EPS Power")?,
-            self.power("p_charge", "Battery Charge Power")?,
```

Comment:
Unrelated change: these are duplicates and their presence makes homeassistant sad.

While working this out, I realised that the retained config messages lxp-bridge sets seem to stick around indefinitely - very annoying when testing. Perhaps that could be reduced somewhat?

Reply:
Hm, I see MQTT v5 has a mechanism for setting a TTL for retained messages, but not MQTT v3.

The MQTT client library crate used by lxp-bridge does seem to support that (https://github.com/bytebeamio/rumqtt/pull/351) but I don't know how widespread server support is, ie if we could safely use it.

The recommended way to remove a retained message is to send an empty payload for the topic with the retain bit *unset* as I understand it, so we could add something in to support that for older discovery topics we no longer want to see.

Reply:
Interesting, thanks.

It's not something we have to be concerned with in this PR, but will be worth remembering in the future if we need to remove or rename one of these.